app/scripts/common/filters/oo-currency.filter.js

```diff
@@ -4,7 +4,10 @@ angular.module('openolitor-core').filter('ooCurrency', ['$rootScope', '$filter',
   function($rootScope, $filter, gettextCatalog, WAEHRUNG, EnumUtil, lodash) {
   return function(value, showTag, currency) {
     var waehrungen = EnumUtil.asArray(WAEHRUNG);
-    var projektCurrency = $rootScope.projekt.waehrung;
+    var projektCurrency = undefined;
+    if ($rootScope.projekt !== undefined && $rootScope.projekt.waehrungen !== undefined ){
+      var projektCurrency = $rootScope.projekt.waehrungen;
```

Comment:
It must be on both cases `$rootScope.projekt.waehrung`.

There is no field `waehrungen` as far as I can see.

Reply:
You are completely right. I do not know how I put accidentally the plural...